core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -374,6 +182,16 @@ export class ContributorAdminStatsTable implements OnInit {
     return contributionCount;
   }
 
+  getFormattedContributionAttributes(
+      element: TranslationSubmitterStats |
```

Comment:
"element" is too vague. Perhaps contributorStats or something like that?

Reply:
Done